### DISCUSSION TITLE
Is it possible to remove or disable locale path /en/

### DISCUSSION THREAD

**Discussion by ghost** [requarks/wiki#3489] [Help / Questions]
Hi, is it possible to disable the locale setting for a site? My wiki.js will only ever be in English and the /en/ path is tripping up search console / analytics as users are sometimes being served the locale paths and other times not, resulting in duplicate pages being logged. Thanks

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

**Comment by thmmsn** [comment]
I solved this by using a reverse proxy in Apache.

In my `wiki.conf` file:

```
<VirtualHost *:80>
 ServerName wiki.example.com
 ServerAlias wiki.example.com
 ProxyPreserveHost on
 ProxyPass / http://<my ip:port>/
 ProxyPassReverse / http://<my ip:port>/
 RewriteEngine On
 RewriteRule ^/XX/(.*)$ http://wiki.example.com/$1 [L,R=301]
</VirtualHost>
```
Where `<my ip:port>` in my case is pointing to `10.0.0.100:3000`
You can change `XX` to whatever language locale you want in `RewriteRule ^/XX/(.*)$ http://wiki.example.com/$1 [L,R=301]`
